src/Microsoft.TemplateEngine.Abstractions/Parameters/ParameterDefinitions.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Abstractions.Parameters
+{
+    /// <inheritdoc />
+    public class ParameterDefinitions : IParameterDefinitionSet
```

Comment:
I wouldn't put it there - it is needed in TemplateSearch.Common (for initialization after deserialization) and it would be the only reason to pull TemplateEngine.Utils as a dependency here